dependencies/bullseye/hammer_cli_foreman_host_reports/upstream/metadata

```diff
@@ -0,0 +1,9 @@
+---
+Archive: GitHub (FIXME)
```

Comment:
What's the FIXME part here?

Reply:
I have no idea what this file is, anyways ;)

Reply:
https://wiki.debian.org/UpstreamMetadata#Fields doesn't tell much and I didn't find an example there :(